trainings/insights/lesson-plan.md

```diff
@@ -0,0 +1,83 @@
+
+
+# Code Insights Webinar
+
+**Topic:** This training is intended as an initial deep-dive to educate users on Code Insights outside of the 101/102 context. **If at all possible, we recommend having the customer provide a Code Insights use case to you *before* the training, giving you the opportunity to work on that specific insight or something similar;** if not, you can use this as a way to show Code Insights to them and discuss use cases they might find valuable live on the training.
+
+## Intro
+
+Today's training is intended to walk users through our Code Insights product, and show users how to use the feature. By the end of it, we hop that you'll feel comfortable with what Code Insights is and can do, and will be ready to start with your own changes in Sourcegraph.
```

Comment:
IMO - I think they should walk away with an understanding of how insights work under the hood. 

So they know what it means if they choose "run over all repos" vs specific repos.

Reply:
Gotcha—do you have more context on how you've seen that work well in practice? (or @Joelkw , if you have thoughts.)

Reply:
@Isuru-F  I leave it up to the CE's judgement on a case by case basis if they think that'd help a customer. What you refer to here – the difference between all repos vs specific repos – is actually going to go away for the most part in 3.42 (you'll see many updates about it in slack next week). (Otherwise, I agree it's usually been valuable in the past to explain.)

Reply:
Ah, gotcha. If that's going away, I'm going to leave this as-is and open it up the CEs to provide feedback on what they ultimately get questions on in the new version, if that works? (And we'll add it in as we get that feedback.)